pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -946,7 +947,7 @@ func TestPodFailedSchedulingMultipleTimesDoesNotBlockNewerPod(t *testing.T) {
 			CreationTimestamp: metav1.Now(),
 		},
 		Spec: v1.PodSpec{
-			Priority: &highPriority,
+			Priority: &lowPriority,
```

Comment:
Change from highPriority to lowPriority does not matters too much, we can keep the highPriority unchanged.
The key point for this UT is that "older Pods will not starve newer Pods, even if the newer Pods has lowPriority". (Of cause, it will also not starve, if the newer Pods has the same or higher Priority)